_docs/_latest/use/cli/scanner.md

```diff
@@ -37,9 +37,14 @@ forseti scanner run
 ```
 
 The external project access scanner is not part of the regular running scanners, as it may take hours to run. 
-Therefore, we now make this scanner accessible manually through the CLI:
+You might also need to create the inventory and model if it is not already
+created. Therefore, we now make this scanner accessible manually through the CLI:
```

Comment:
I think it reads better this way - 

Therefore, we now make this scanner accessible manually through the CLI:

Note: You will need to create an inventory and model if it is not already created.